lib/models/user_model.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
```

Comment:
Can you use the class structure defined at https://onedrive.live.com/view.aspx?resid=D555577229855EE4%211149&id=documents&wd=target%28Database.one%7C43178699-7208-BE42-8EB6-C6E9D52C1C4A%2FModels%7C472A3342-4FE3-774E-9988-85EFDE8D642F%2F%29
onenote:https://d.docs.live.net/d555577229855ee4/Documents/Badi%20App/Database.one#Models&section-id={43178699-7208-BE42-8EB6-C6E9D52C1C4A}&page-id={472A3342-4FE3-774E-9988-85EFDE8D642F}&end

Reply:
Oops, sorry I forgot to check our data structure,
I will update it.